src/main/java/DukeSession.java

```diff
@@ -0,0 +1,92 @@
+import file.methods.FileHandler;
+import utility.Methods;
+import utility.commandChecker;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+
+import java.io.File;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
+public class DukeSession {
+    private ArrayList<Task> actions = new ArrayList<>();
+    public void setUpArrayList() {
+        FileHandler.setUpFile(actions);
+    }
+    public void execute() {
```

Comment:
This method is quite long (>30 lines). Consider refactoring the code to a new method OR see how you can combine common things from the case statements together. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)